apps/extension/webpack.config.js

```diff
@@ -112,7 +112,15 @@ export default (env, argv) => {
       }),
       new webpack.DefinePlugin(definitions),
       new CopyPlugin({
-        patterns: ['public', { from: 'bin', to: 'bin' }],
+        patterns: [
+          { from: 'public', to: '.' },
+          {
+            from: 'bin',
+            to: 'bin',
+            context: path.resolve(__dirname, '../../packages/wasm'),
+            noErrorOnMissing: true,
+          },
+        ],
```

Comment:
iirc there is a more 'conventional' way to bundle assets from a dependency in webpack. i don't have an example on hand but i will see what i can find later